configuration/ConfigHandler.cpp

```diff
@@ -0,0 +1,50 @@
+#include "ConfigHandler.hpp"
+#include <iostream>
+#include <fstream>
+#include "../yaml-cpp/include/yaml-cpp/yaml.h"
+
+Configuration::ConfigHandler::ConfigHandler(std::string filePath)
+{
+    this->getConfigFile(filePath);
+}
+
+Configuration::ConfigHandler::~ConfigHandler(){}
+
+void Configuration::ConfigHandler::getConfigFile(std::string configFile)
```

Comment:
Please make each entry getter more generic, currently if we want to add other fields we have to add multiples lines to this one function, which will make the code unmaintainable.